lib/Sema/CodeSynthesisDistributedActor.cpp

```diff
@@ -33,47 +33,10 @@
 
 using namespace swift;
 
-
 /******************************************************************************/
 /************************ PROPERTY SYNTHESIS **********************************/
 /******************************************************************************/
 
-static VarDecl*
-lookupDistributedActorProperty(NominalTypeDecl *decl, DeclName name) {
```

Comment:
This method was duped, we have it already 
```
swift::lookupDistributedActorProperty(NominalTypeDecl *decl, DeclName name) {
```

with the same impl, so we dont need it here